### PR TITLE
feat: Release notes for Codacy Cloud January 2023 DOCS-437

### PR DESCRIPTION
**Pull request by prcr**
Adds auto-generated release notes for Codacy Cloud January 2023

### :eyes: Live preview
https://feat-release-notes-cloud-2023-01--docs-codacy.netlify.app/release-notes/cloud/cloud-2023-01/

### :construction: To do
- [x] Add the new release notes page to `mkdocs.yml` and `docs/release-notes/index.md`
- [x] Review generated release notes
- [x] Review list of issues with missing release notes
- [x] Review links to changelogs of updated tools
- [x] Update release date and remove `TODO` comments

### CODE REVIEW

File: docs/release-notes/cloud/cloud-2023-01.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+-   Codacy now supports [reporting a summary of the coverage metrics](../../repositories-configure/integrations/github-integration.md#enabling) directly on your pull requests. For now, this feature is only supported on GitHub Cloud. (COV-2, COV-3)
+
+    ![Coverage summary on GitHub](../images/cov-2.png)
+
+## Bug fixes
+
+-   Codacy no longer counts ignored files towards the [maximum number of files for calculating complexity](../../faq/code-analysis/does-codacy-place-limits-on-the-code-analysis.md). (TS-93)
+-   Fixed an issue that caused all ESLint 8 code patterns to be turned off by default when adding a new repository. (TS-50)
+-   Fixed the repository page not opening for Bitbucket repositories with many branch restrictions and repository:admin users. (PLUTO-290)
```
Comment (prcr) [inline docs/release-notes/cloud/cloud-2023-01.md:34]: @claudiacarpinteiro please push a commit to this pull request branch with any edits that you decide to do on this release note. :rocket:

**Review by fjrdomingues — APPROVED** [review]
The coverage parts look great 👍

File: docs/release-notes/cloud/cloud-2023-01.md (hunk 1 above)
Comment (prcr) [reply docs/release-notes/cloud/cloud-2023-01.md]: I pushed another commit just to add the "code" style to the `repository:admin` scope. :nerd_face:

File: docs/release-notes/cloud/cloud-2023-01.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# Cloud January 2023
+
+These release notes are for the Codacy Cloud updates during January 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The Codacy Coverage Reporter now supports [automatic commit SHA-1 detection](../../coverage-reporter/troubleshooting-coverage-cli-issues.md#commit-detection) on Argo CD. (TS-140)
+-   Extended the visibility of the [coding standards page](../../organizations/using-a-coding-standard.md) to all users, regardless of their permission level. (IO-359)
```
Comment (prcr) [inline docs/release-notes/cloud/cloud-2023-01.md:19]: @andre-a-pires Please validate that this update is good for including on our January release notes.

File: docs/release-notes/cloud/cloud-2023-01.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+-   The pull request detail page now displays the [commits that are missing coverage data](../../coverage-reporter/index.md#validating-coverage) for Codacy to calculate the coverage metrics. This information is also visible in the analysis logs. (IO-152)
+
+    ![Pull request analysis logs showing the commits that are missing coverage data](../images/io-152b.png)
+
+    <!--NOTE This Jira key appears on two release note pages because the feature had an update after being released (see https://codacy.atlassian.net/browse/COV-137)-->
+
+-   The documentation now includes an end-to-end example illustrating the usage of an [API endpoint to identify commits without coverage data](../../codacy-api/examples/identifying-commits-without-coverage-data.md) and troubleshoot issues with the calculation of coverage metrics. (DOCS-503)
```
Comment (nicklem) [inline docs/release-notes/cloud/cloud-2023-01.md:26]: @claudiacarpinteiro Ready for your review!

Comment (claudiacarpinteiro) [inline docs/release-notes/cloud/cloud-2023-01.md:26]: Looks great! 👍 
Small suggestion, if it makes sense to you:

```suggestion
-   The documentation now includes an end-to-end example illustrating the usage of an [API endpoint to identify commits without coverage data](../../codacy-api/examples/identifying-commits-without-coverage-data.md) to troubleshoot issues with the calculation of coverage metrics. (DOCS-503)
```

**Review by claudiacarpinteiro — APPROVED** [review]
LGTM

File: docs/release-notes/cloud/cloud-2023-01.md (hunk 2 above)
Comment (nicklem) [reply docs/release-notes/cloud/cloud-2023-01.md:19]: Resolving this one to merge. @andre-a-pires feel free to ping any of us tech writers if needed!

**Comment by andre-a-pires** [comment]
side question: why do we share our internal jira issues code (ex: IO-123) if they're not public?

**Review by andre-a-pires — COMMENTED** [review]
Looks good 👍

**Comment by nicklem** [comment]
@andre-a-pires I'm not 100% sure about the rationale, but it's likely for future reference, should the need arise.

If, for instance, a customer complains about a feature we mentioned in the release notes, it's easier to find our way back to the task, commits, discussions, and so on. We try to have as many links as possible to minimize time spent looking stuff up.

cc: @prcr

**Pull request merged**